index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+      <a id="profile-link" href="https://codepen.io/pen?template=MJjpwO"
+         >My profile
+      </a>
+      <a class="links" href="#projects">Projects</a>	
+      <a class="links" href="#more">More</a>
+    </nav>
+   
+     <!-- welcome section -->
+   
+     <section id="welcome-section">
+       <h1>Hey guys...</h1>
+       <p>Lover of HTML and CSS</p>
+     </section>
+   
+     <!-- projects -->
+   
+     <section id="projects">
+        <a href="https://codepen.io/ks-97/pen/OJgqRjm">
+        <div class="project-tile">
+            <img src="https://images.unsplash.com/photo-1533749047139-189de3cf06d3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80"
+            alt="clock">
+            <p>
+            Project on a digital clock.
+            </p>
+        </div>
```

Comment:
Please use semantic tags. A figure and figcaption elements would prove useful. Avoid generic tags as much as possible.

Reply:
- [ ] Instead, you can write "Kindly use semantic tags more following the HTML best practices. Try using generic tags less as it's not recommended more often."

Reply:
Kindly use semantic tags more often than generic tags following the HTML best practices.